infrastructure/helm-chart/charts/provisioning/templates/scripts.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: provisioning-scripts
+data:
+  wait-for-service.sh: |
+    #!/bin/sh
+    service=${1}
+    port=${2}
+    delay=${3}
+    name=${4}
+
+    while ! nc -z ${service} ${port} >/dev/null 2>&1; do sleep ${delay}; echo Waiting for ${name} to start...; done
```

Comment:
A general helper function, for waiting for services, to be used in `init containers`.